moon-store-mac-os/moon-store-mac-os/Views/User/UserListView.swift

```diff
@@ -0,0 +1,204 @@
+        static let plusIcon: String = "paperplane.fill"
+    }
+    
+    enum UserRow {
+        static let spacing: CGFloat = 10
+        static let editTitle: String = "Edit"
+        static let trashIcon: String = "trash"
+        static let iconSize: CGFloat = 20
+        static let hStackSpacing: CGFloat = -30
+        static let lineLimit: Int = 1
+        static let evenNumber: Int = 2
+    }
+    
+    enum Alert {
+        static let iconSize: CGFloat = 16
+        static let title: String = "Dirección de correo:"
+        static let buttonTitle: String = "Invitar"
+        static let iconTitle: String = "xmark.circle.fill"
+        static let placeholder: String = "example@example.com"
+        static let height: CGFloat = 120
+        static let verticalpadding: CGFloat = 5
+        static let buttonHeight: CGFloat = 100
+    }
+}
+
+enum UserTableHeader: CaseIterable {
+    case icon, user, email, role, date, option
+    
+    var title: String {
+        switch self {
+            case .user: "Usuario"
+            case .email: "Email"
+            case .role: "Rol"
+            case .date: "Fecha"
+            default: ""
+        }
+    }
+}
+
+struct UserListView: View {
+    @State private var showActionSheet: Bool = false
+    
+    var body: some View {
+        VStack {
+            HeaderView(
+                title: Constants.headerTitle,
+                icon: Constants.personIcon
+            )
+            
+            HStack {
+                Text(Constants.userTitle)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                    .font(.title3)
+                    .bold()
+                
+                Button {
+                    showActionSheet.toggle()
+                } label: {
+                    Label(Constants.Button.title,
+                          systemImage: Constants.Button.plusIcon)
+                    .foregroundStyle(.msWhite)
+                }
+                .buttonStyle(.plain)
+                .padding(Constants.padding)
+                .background(.msPrimary, in: .rect(cornerRadius: Constants.cornerRadius))
+            }
+            
+            productTableView
+        }
+        .frame(maxWidth: .infinity, alignment: .top)
+        .padding(.horizontal)
+        .sheet(isPresented: $showActionSheet) {
+            UserInviteView(isShowing: $showActionSheet)
+        }
+    }
+    
+    // MARK: - Table View
+    
+    private var productTableView: some View {
+        VStack(spacing: .zero) {
+            headerTableView
+                .background(.msWhite)
+            
+            ScrollView(showsIndicators: false) {
+                Grid(horizontalSpacing: .zero, verticalSpacing: .zero) {
+                    ForEach(Array(UserModel.userMockData.enumerated()), id: \.element.id) { index, user in
+                        HStack(spacing: Constants.UserRow.hStackSpacing) {
+                            productRowView(
```

Comment:
Esto deberia ser renombrado por `userRowView`

```suggestion
                            userRowView(
```